Week3/script.js

```diff
@@ -72,24 +72,22 @@ function threeArguments(a, b, c) {
 
         // 2
 
-function colorCar(car) {
-  return car;
+function colorCar() {
+  return "a red car";
 }
 
-let car = "a red car";
-
-console.log(colorCar(car));
+console.log(colorCar());
 
         // 3
 
 const person = {
-  name: "Mehmet Ekinci",
-  gender: "M",
-  adres() {
-    console.log("Koemeershoeve");
-  }
+  name: 'Mehmet',
 };
-console.log(person);
+
+function personInfo() {
+  return `His ${Object.keys(person)} is ${person.name}`;
```

Comment:
Actually I think just `console.log(person)` is fine. There is a nicer way of doing it, but for now this is fine:).

To read more about ways to do this, you can read [this](https://zellwk.com/blog/looping-through-js-objects/) (and if you want, you can change your code;)